python/mlc_llm/serve/server/server_context.py

```diff
@@ -14,6 +15,7 @@ class ServerContext:
 
     _models: Dict[str, async_engine.AsyncThreadedEngine] = {}
     _conv_templates: Dict[str, Conversation] = {}
+    _model_config_paths: Dict[str, str] = {}
```

Comment:
Let's load the model config json and save the config dictionary in ServerContext. So we don't need to parse and read the JSON every time in the entrypoint.

Reply:
Yes that makes sense. Changed.